.github/workflows/sync.yml

```diff
@@ -0,0 +1,51 @@
+name: Daily list sync
+
+on:
+  schedule:
+    - cron: "30 15 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Elixir
+        uses: erlef/setup-elixir@v1
+        with:
+          otp-version: "22.2"
+          elixir-version: "1.9.4"
```

Comment:
Do we really need Elixir here? Sounds like we're only using bash/git commands

Reply:
Fixed.